scripts/docker-smoke-test.sh

```diff
@@ -38,18 +48,42 @@ await_server_ready() {
 
 stop_container() {
     echo Stopping container "$CONTAINER_NAME"
+    # Note (austin) - if you're getting an error from the api, try dumping the logs
+    # docker logs $CONTAINER_NAME 2> docker_logs.txt
     docker stop "$CONTAINER_NAME"
 }
 
-start_container
-
-# Regardless of test result, stop the container
+# Always clean up the container
 trap stop_container EXIT
 
+start_container "false"
 await_server_ready
 
-echo Running tests
+#######################
+# Smoke Tests
+#######################
+echo Running smoke tests
 PYTHONPATH=. SKIP_INFERENCE_TESTS=$SKIP_INFERENCE_TESTS pytest scripts/smoketest.py
 
+#######################
+# Test parallel vs single mode
+#######################
+echo Running parallel mode test
+
+curl http://localhost:$API_PORT/general/v0/general --form 'files=@sample-docs/layout-parser-paper.pdf' --form 'coordinates=true' --form 'strategy=fast' | jq -S > single-mode.json
+
+stop_container
+start_container "true"
+await_server_ready
+
+curl http://localhost:$API_PORT/general/v0/general --form 'files=@sample-docs/layout-parser-paper.pdf' --form 'coordinates=true' --form 'strategy=fast' | jq -S > parallel-mode.json
```

Comment:
also test `strategy=hi_res` ?

Reply:
another test for `coordinates=false`, or not provided?

Reply:
Ah, good call. I'll have this test loop through a number of curl commands.